.github/workflows/build.yml

```diff
@@ -257,7 +257,7 @@ jobs:
         id: pack_artifacts
         if: ${{ ( github.event_name == 'push' && github.ref == 'refs/heads/master' ) || github.event.inputs.create_release == 'true' }}
         run: |
-          Copy-Item ..\LICENSE .\bin\Release\llama.cpp.txt
+          Copy-Item LICENSE .\build\bin\Release\llama.cpp.txt
```

Comment:
lol

Reply:
Me being pedantic about licenses coming back to bite me :)